packages/gatsby-theme-docs/src/components/card.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import css from '@emotion/css';
+import { navigate } from '@reach/router';
+import SpacingsStack from '@commercetools-uikit/spacings-stack';
+import SpacingsInline from '@commercetools-uikit/spacings-inline';
+import { designSystem } from '@commercetools-docs/ui-kit';
+// import markdownFragmentToReact from '../utils/markdown-fragment-to-react';
```

Comment:
@emmenko I'm getting an error that prevents the card component from being exported when I import `markdownFragmentToReact`, can you please help with this?

Reply:
Can you be more specific? What error?

Reply:
it's logged in the browser:

![Screenshot 2020-05-20 at 11 16 25](https://user-images.githubusercontent.com/8238495/82428969-86bebc00-9a8b-11ea-9e52-0d21002b586b.png)

![Screenshot 2020-05-20 at 11 16 59](https://user-images.githubusercontent.com/8238495/82428995-8e7e6080-9a8b-11ea-8716-880f96a633df.png)

Reply:
Is this happening only when you import `markdownFragmentToReact`?

How are you using that function?

Reply:
it happens only when I import.

Reply:
Just a guess maybe. I'm not sure this is supposed to work like this to be honest.
The card components are rendered within MDX, and here we're trying to render yet another MDX within MDX...it kind of feels like a vicious loop.

I suppose we just need to let MDX render the content, without us needing to interfere.

Again, I haven't looked into that yet, just some long guess.

@nkuehn what do you think?

Reply:
@nkuehn found the problem :)

Reply:
@emmenko cyclic import -> export -> import of the same thing.  never thought about that being an issue.  The card was imported into the fragement renderer via the MDX markdown components and then imported by the card.  

It wasn't correct from the beginning to have the markdown fragment renderer import the basic UI kit components via the MDX components provider instead of directly.  

Concerning "do we need this at all" -> yes, I'm running into more and more cases where it's becoming clear that you need different rendering if it's "squeezed inside some other box".   E.g. just opened #451